tests/phpunit/CRM/Utils/TokenConsistencyTest.php

```diff
@@ -234,6 +234,45 @@ public function getUnadvertisedTokens(): array {
     ];
   }
 
+  /**
+   * Test tokens in 2 ways to ensure consistent handling.
+   *
+   * 1) as part of the greeting processing
+   * 2) via the token processor.
+   *
+   */
+  public function testOddTokens(): void {
+
+    $variants = [
+      [
+        'string' => '{contact.individual_prefix}{ }{contact.first_name}{ }{contact.middle_name}{ }{contact.last_name}{ }{contact.individual_suffix}',
+        'expected' => 'Mr. Anthony  Anderson II',
```

Comment:
OK, so your hesitation is that this requires a double-space? Which would be invisible on most HTML outputs - but visible on text outputs?

I think if this produces the same output on 5.42, then I wouldn't worry about it.

Reply:
thanks @totten